docs/guides/server/bootstrap-admin-recovery.adoc

```diff
@@ -0,0 +1,90 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Admin bootstrap and recovery"
+summary="Learn how to bootstrap and recover admin account." >
+
+== A solution for a disaster scenario
+
+It used to be difficult to regain access to a {project_name} instance, where all admin users have been locked out. The process required multiple user-advanced steps, including direct database access and manual changes. In the continuous effort to improve the user experience, {project_name} now provides multiple ways to bootstrap a new admin account, which can be later used to recover from such a situation.
```

Comment:
This paragraph is good but possibly should be moved to release notes. IMHO guides should not reminiscence on some previous state of things.

Instead, I'd suggest some introduction that would explain the user the concept of temporary admin user (that they should create a more permanent and secure user etc.).